src/sd.c

```diff
@@ -311,14 +312,17 @@ void sd_free_list(sd_list_t* list)
 
 bool sd_card_inserted(void)
 {
+    // The sd card is allowed up to 1s to initialize, therefore the usb processing timeout must also
+    // be increased by about 1s.
+    usb_processing_timeout_reset(-10);
```

Comment:
Did you check if we actually sleep in case of backup create, assuming the app checked sd card presence beforehand once (which it does)? So we can more confidently release a hotfix knowing it's the right fix. Maybe one of the other operations in backup creation (copy_seed, writing backup) also causes delays and we should give more time there?

Reply:
during the device initialization it didn't sleep during the backup creation for me. But I don't think that is conclusive either way.